app/src/components/DataPanel.vue

```diff
@@ -54,6 +49,17 @@
               @update:comparelayertime="c => comparelayertime = c"
               @compareEnabled="compareEnabled = !compareEnabled"
             /> -->
+            <v-btn
+              ref="regenerateButton"
+              color="secondary"
+              style="display: block;"
```

Comment:
the button is currently positioned on the left, could it be positioned next to the "reset zoom" button? something like `position: absolute; right: <...>px; top: 13px;`